lib/puppet/provider/package/pecl.rb

```diff
@@ -20,19 +20,17 @@ def self.pecllist(hash)
     begin
       list = execute(command).split("\n").map do |set|
         if hash[:justme]
-          if %r{^#{hash[:justme]}$}i.match(set)
+          if %r{^#{hash[:justme]}$}i =~ set
             if peclhash = peclsplit(set) # rubocop:disable Lint/AssignmentInCondition
```

Comment:
Good spot.  It certainly looks like it, doesn't it?  Lets look at the original code before any rubocop and refactoring started... https://github.com/voxpupuli/puppet-php/blob/4.0.0-beta1/lib/puppet/provider/package/pear.rb#L29

So assuming https://github.com/voxpupuli/puppet-php/pull/240/commits/6bbdc78177b53b84b6e6e7f27eb0edb21309e6ae was correct and `rubocop --auto-correct --only Style/EmptyElse` wasn't mistaken, then yes to merging the conditionals?

It's a real shame there are no spec tests covering this code...